README.md

```diff
@@ -219,6 +229,7 @@ Some default Python arguments are provided in case the user is using the default
 * `--memory [amount]`: amount of memory to request for job [MB] (default = 2000)
 * `--cpus [number]`: number of CPUs (threads) for job (default = 1)
 * `--sites [list]`: comma-separated list of sites for global pool running (if using CMS Connect) (default from `.prodconfig`)
+* `--env [args]`: run in Singularity environment using cmssw-env (default = None)
```

Comment:
I know it specified the default, but is there a way, either here or near line 191, to give an example argument? What would a person specify to make this run?

Reply:
Right now it's just the name/path of the container. I'll add an example at L191.

Reply:
Seeing the example now, can't any CVMFS hosted and flattened singularity sandbox be used? I mean, I guess there are a few more requirements based on step1.sh, but assuming those are satisfied... Maybe you want to reword L191 to:
> Existing CMSSW containers available at `/cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/`, but other CVMFS hosted Singularity sandboxes can also be used.